src/middleware/auth/user/authDataOnlyMiddleware.ts

```diff
@@ -12,18 +9,14 @@ const authDataOnlyMiddleware = async (req: Request, res: Response, next: Next) =
 
   if (userAccessToken == null) return res.sendStatus(401);
 
-  const TOKEN = process.env.ACCESS_TOKEN_SECRET;
-
   let user;
 
-  if (TOKEN === undefined) console.log("No secret access token in env");
-  else
-    try {
-      user = jwt.verify(userAccessToken, TOKEN);
-    } catch (err) {
-      console.log(err);
-      return res.sendStatus(403);
-    }
+  try {
+    user = jwt.verify(userAccessToken, process.env.JWT_SECRET_USER);
+  } catch (err) {
+    console.log(err);
+    return res.sendStatus(403);
```

Comment:
Better to send an error to express (`next(error)`). You can use this to generate nice errors: https://www.npmjs.com/package/http-errors